packages/wrangler/bin/wrangler.js

```diff
@@ -3,7 +3,6 @@ const { spawn } = require("child_process");
 const path = require("path");
 const fs = require("fs");
 const os = require("os");
-const semiver = require("semiver");
 
 const MIN_NODE_VERSION = "16.7.0";
```

Comment:
Could we read MIN_NODE_VERSION from package.json?

Reply:
we'd have to parse it out because that won't be a straight version number, don't want to add that here